src/nft/components/details/AssetPriceDetails.tsx

```diff
@@ -372,7 +372,7 @@ export const AssetPriceDetails = ({ asset, collection }: AssetPriceDetailsProps)
                     }
                   }}
                 >
-                  <SubHeader lineHeight={'20px'}>
+                  <SubHeader color="white" lineHeight={'20px'}>
```

Comment:
should this be primaryColor?

Reply:
primary isn't white in light theme

Reply:
@aballerr could you sync with yushin on this? Apparently they defined a `button-label` in the design system that reflects what the text colors should be for light and dark.

Reply:
![Screen Shot 2022-11-14 at 11 03 31 AM](https://user-images.githubusercontent.com/15934595/201707531-2824d067-3560-41d2-b964-3af4abe4167a.png)
![Screen Shot 2022-11-14 at 11 03 38 AM](https://user-images.githubusercontent.com/15934595/201707534-3e403be3-1a89-44e1-b5b5-ec696176662e.png)


This is what buy now currently looks like and this is what buttons on our site look like.  I'm not really sure what I need to sync on to be honest